files/photoswipe/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "PhotoSwipe",
+  "repo": "dimsemenov/PhotoSwipe",
+  "files": {
+    "basePath": "dist/"
```

Comment:
How about doing `include "*.min.*", "photoswipe.css", "./default-skin/**/*"`?
This way we host only the minified files